constraint/config/crds/templates.gatekeeper.sh_constrainttemplates.yaml

```diff
@@ -65,6 +65,26 @@ spec:
               targets:
                 items:
                   properties:
+                    code:
+                      description: The source code options for the constraint template.
+                        "Rego" can only be specified in one place (either here or
+                        in the "rego" field)
+                      items:
+                        properties:
+                          engine:
+                            description: 'The engine used to evaluate the code. Example:
+                              "Rego". Required.'
+                            type: string
+                          source:
+                            description: The source code for the template. Required.
+                            x-kubernetes-preserve-unknown-fields: true
+                        required:
+                        - engine
```

Comment:
Should `source` be required too? It says required in the description.

Reply:
good catch, apparently "omitempty" overrides the controller-gen flag, fixed.